src/components/raycaster.js

```diff
@@ -31,6 +31,15 @@ module.exports.Component = registerComponent('raycaster', {
     this.prevCheckTime = undefined;
     this.raycaster = new THREE.Raycaster();
     this.updateOriginDirection();
+    this.refreshObjects = this.refreshObjects.bind(this);
+  },
+
+  play: function () {
+    this.el.sceneEl.addEventListener('child-attached', this.refreshObjects);
```

Comment:
We can optimize later. I need this patch for the a-painter and don't have the bandwidth right now to do something more elaborate.